atmo/scheduling.py

```diff
@@ -98,3 +98,21 @@ def spark_job_run(user_email, identifier, notebook_uri, result_is_public, size,
         VisibleToAllUsers=True,
     )
     return cluster['JobFlowId']
+
+
+def get_spark_job_results(identifier, is_public):
+    if is_public:
+        bucket = settings.AWS_CONFIG['PUBLIC_DATA_BUCKET']
+    else:
+        bucket = settings.AWS_CONFIG['PRIVATE_DATA_BUCKET']
+    results = dict(data=[], logs=[])
+
+    s3_objects = s3.list_objects(Bucket=bucket, Prefix='{}/'.format(identifier))
+    if 'Contents' in s3_objects:
+        for item in s3_objects['Contents']:
+            try:
+                prefix = item['Key'].split('/')[1]
```

Comment:
I don't think so. Those directories are not created by the notebook itself, but by the bootstrap script. A notebook could potentially create subdirs of data, but that should be a problem.